components/wpt-runs.html

```diff
@@ -136,7 +136,7 @@
       }
 
       _computeDateTooltip (date) {
-        return date.toDateString()
+        return date && date.toDateString()
```

Comment:
What's this for? I see nothing related to _computeDateTooltip and nothing related in the commit message.

Reply:
When the computation occurs on bad data, this just prevents an exception.